Generator/source/generatorAPI.cpp

```diff
@@ -0,0 +1,349 @@
+#include <map>
+#include <array>
+#include <chrono>
+#include <vector>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include <algorithm>
+#include <filesystem>
+
+#include <unistd.h>
+#include <getopt.h>
+
+#include "nlohmann/json.hpp"
+
+#include <AuxiliaryMethods/AuxiliaryMethods.h>
+#include <DataBase/DataBaseGenerator.h>
+#include <generators/GenerationParameters.h>
+#include <DataBase/DataBaseGeneratorParameters.h>
+
+#include <generatorAPI.h>
+
+using namespace std::chrono;
+
+void runGenerationFromJson(std::string json_path)
+{
+  std::ifstream f(json_path);
+  nlohmann::json DATA = nlohmann::json::parse(f);
+  // Read all json objects in json file.
+  for (auto it = DATA.begin(); it != DATA.end(); it++)
+  {
+    nlohmann::json data = *it;
+
+    // Проверка на наличие типа генерации. Если его нет, то завершаем программу.
```

Comment:
Previous comment was in English (line 29), here Russian language is used -- so what should we select?

Reply:
Later.